### ISSUE TITLE
The setting of midea_ac_lan.set_attribute fan_speed is not working.

### ISSUE THREAD

**Issue by v1k70rk4** [georgezhao2010/midea_ac_lan#327]
### HA core version

2023.9.2

### Intergration version

0.3.21

### Device type and model

Air Conditioner 00000Q11, Air Conditioner 00000Q19

### Used App

MSmartHome

### The description of problem

Hello, I have been using the standalone Fan_Speed service until now, and it has been discontinued. I have now switched to this mode:
```
service: midea_ac_lan.set_attribute
data:
  device_id: 144036023355376
  attribute: fan_speed
  value: 40
```
Unfortunately, this does nothing. Nothing appears in the Debug log either, even though the command runs successfully.

If I use the `attribute: eco_mode` part, the service adjusts that setting, but fan_speed does not work.

### The logs

Unfortunately, nothing appears even in debug mode.

**Comment by georgezhao2010** [comment]
> ### The description of problem
> Hello, I have been using the standalone Fan_Speed service until now, and it has been discontinued. I have now switched to this mode:
> 
> ```
> service: midea_ac_lan.set_attribute
> data:
>   device_id: 144036023355376
>   attribute: fan_speed
>   value: 40
> ```
> 
> Unfortunately, this does nothing. Nothing appears in the Debug log either, even though the command runs successfully.
> 
> If I use the `attribute: eco_mode` part, the service adjusts that setting, but fan_speed does not work.

Fixed in https://github.com/georgezhao2010/midea_ac_lan/commit/e47cb4169ad7eaa10247b918769428fc2f195ea1.

You can try and see.

**Comment by v1k70rk4** [comment]
> Fixed in [e47cb41](https://github.com/georgezhao2010/midea_ac_lan/commit/e47cb4169ad7eaa10247b918769428fc2f195ea1).
> 
> You can try and see.

Thanks for the super quick hotfix; it works perfectly.
